projects/raft-otel/README.md

```diff
@@ -97,50 +97,41 @@ Read these blog posts carefully.
-This simply appends the `command` provided to a log.
-
-**Note:** What Bendersky is doing here is rather sketchily demonstrating a theoretical [Finite State Machine](https://en.wikipedia.org/wiki/Finite-state_machine) (FSM).
-FSMs are a computer science concept (that you don't need to know any details about for this project): the idea that you can implement a program's core state as an abstract machine, with a specific set of states
-and a specific set of transitions between those states. Externally-provided commands are the trigger to move between states.
-
-Try modifying your code to instead implement Get, Set, and CompareAndSet commands - a highly-consistent key-value store - and make these available via gRPC methods.
-For instance, a Set command may have two fields, a key and a value, and a Get command may have one field, a key. If your RAFT cluster accepts a Set command setting X = 10, then a Set command setting X = 20, and then gets a Get command for X, it should return 20.
-There is already a `storage` module that you can use.
-
-**Note:** [CompareAndSwap](https://en.wikipedia.org/wiki/Compare-and-swap) (also called CompareAndSet, or CAS)
+[CompareAndSwap](https://en.wikipedia.org/wiki/Compare-and-swap) (also called CompareAndSet, or CAS)
 is a very useful pattern for concurrent systems that lets you update a key to a given value only if that key already has a specific value.
 This is useful for implementing sequences of operations where we read a value, perform some computation that modifies that value, and then write that value back -
 but without potentially overwriting any changes to that value that other processes might have performed.
 
-Next, write a client that uses your RAFT cluster to perform Sets, Gets, and CompareAndSets.
-Bendersky's code also doesn't do anything if you send a `Submit` to any server other than the leader. It may be useful to have your program return the
-address (host:port) of the leader instead, as part of your gRPC reply, along with an indication that the operation was not attempted. Your client can then
-retry the operation against the leader.
+Update the demo client to use the new operation.
+
+##### Operations when an instance is not the leader - redirecting or proxying clients
+
+The initial code  doesn't do anything if you send a get/set to any server other than the leader. It may be useful to have your program return the
+address (host:port) of the leader instead, as part of your gRPC reply. Your client can then
+retry the operation against the leader. Alternatively, you could modify the program to proxy the request directly to the leader.
 
 Please timebox this part of the project to no more than two days, in order to leave time for the other sections.
 At the start of the third day, if your implementation is not complete, begin modifying Bendersky's. You can come back and complete your
 own implementation if you have time at the end of the sprint.
```

Comment:
Is this block meant to apply to this feature? It feels like at least the "Stop implementing your own raft" is misplaced here, maybe the entire para is?

Reply:
mvoed this to top.